human/time_test.go

```diff
@@ -0,0 +1,130 @@
+package human
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+)
+
+func TestTimeParse(t *testing.T) {
+	now := time.Now()
+
+	for _, test := range []struct {
+		in  string
+		out Duration
```

Comment:
Should these tests be comparing with `Time` instead of `Duration`?

Reply:
The test adds the duration to `now` (captured above). I did it this way to avoid having to repeat the code in each test declaration.

Let me know if you think it's too confusing tho, I can change it.

Reply:
Now that I know it isn't just copy-pasta, I'll look over it again it might make more sense with that context. :)